src/tests/ftest/pool/ConnectTest.py

```diff
@@ -82,47 +58,45 @@ def test_connect(self):
         # if any parameter results in failure then the test should FAIL
         expected_result = 'PASS'
         for result in expected_for_param:
-                if result == 'FAIL':
-                      expected_result = 'FAIL'
-                      break
+            if result == 'FAIL':
+                expected_result = 'FAIL'
+                break
         try:
-                uid = os.geteuid()
-                gid = os.getegid()
+            uid = os.geteuid()
+            gid = os.getegid()
 
-                # TODO make these params in the yaml
-                daosctl = self.basepath + '/install/bin/daosctl'
+            # TODO make these params in the yaml
```

Comment:
(pylint-fixme) TODO make these params in the yaml

Reply:
Pre-existing TODO only highlighted here due to whitespace fix on the line.